src/dft/test/regression_tests.tcl

```diff
@@ -1,7 +1,7 @@
 record_tests {
   one_cell_sky130
   one_cell_nangate45
-  sub_modules_sky130
+  #sub_modules_sky130
```

Comment:
why removed?

Reply:
because it is a hierarchical case which has a hierarchical result in the new flow (so the old wrong result should be skipped for now).